gulpfile.js

```diff
@@ -129,7 +131,21 @@ defineBuildTask('HTMLImports');
 defineBuildTask('ShadowDOM');
 defineBuildTask('MutationObserver');
 
-gulp.task('build', ['webcomponents', 'webcomponents-lite', 'CustomElements', 
+gulp.task('CustomElementsV1', function () {
+  return gulp.src('./src/CustomElements/v1/CustomElements.js', {base: './'})
+      .pipe(closureCompiler({
```

Comment:
Justin is correct that Closure does still provide a few optimizations other tools don't (see https://github.com/samccone/The-cost-of-transpiling-es2015-in-2016 for one comparison). 

I am concerned about having a separate build process and pipeline to the rest of the project potentially confusing contributors. I'd love to see us either rally around closure as other parts of the repo get updated (e.g SD V1) or better integrating this work with the existing build system.